src/test/java/uk/gov/companieshouse/api/accounts/ApplicationConfigurationTest.java

```diff
@@ -9,25 +9,25 @@
 
 import uk.gov.companieshouse.environment.EnvironmentReader;
 
-public class ApplicationConfigurationTest {
+class ApplicationConfigurationTest {
```

Comment:
Do we know the default if public isn't specified? (Was this private before?)

Reply:
From what I can see it's package-private

Reply:
awesome

Reply:
Yes, package private by default if unspecified in Java.